.github/workflows/integration-test.yml

```diff
@@ -12,28 +12,53 @@ env:
 jobs:
   tests:
     name: Run integration tests
-    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest , windows-latest ]
+        jdk: [ 11, 17 ]
+    runs-on: ${{ matrix.os }}
+
     steps:
       - name: Checkout Branch
-        uses: actions/checkout@v2
+        uses: actions/checkout@v3
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.jdk }}
 
       - uses: browser-actions/setup-geckodriver@latest
```

Comment:
We can still use commit ref-id directly, I found this from main today from that action
```suggestion
      - uses: browser-actions/setup-geckodriver@c3570fb
```

Reply:
I also tried the commit id earlier today and it doesn't work as we expected :(

<img width="863" alt="Screenshot 2023-01-31 at 8 45 43 PM" src="https://user-images.githubusercontent.com/109499885/215952896-d9dda37a-7076-4835-a5c0-ee17584cf35f.png">

Reply:
Please look into this, it is insecure to reference a branch name.

Reply:
I just switched it to `v0.0.0` since this is the only version they have for now: https://github.com/marketplace/actions/setup-geckodriver?version=v0.0.0#setup-geckodriver